src/configManager/helpers/configDataMap.js

```diff
@@ -1,4 +1,6 @@
 const constConfig = require('../../common/consts').CONFIG;
+const packageJson = require('../../../package');
+const RUNNER_VERSION = packageJson.version.substring(0, packageJson.version.length - 2);
```

Comment:
`const { version: PREDATOR_PLATFORM_VERSION } = require('../../../package');` is more elegant imo